### PR TITLE
include angular and electron frameworks

### PR DESCRIPTION
**Pull request by Greenxertz**
# Description
included the content for angular framework, and included a framework I came across of turning a site into a desktop application. Discord actually uses it. 

Fixes # (issue)
 corrected some title levels
## Type of change

- [x] Document Change (addition or subtraction to our documentation)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] My code/documents follow the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,27 +179,27 @@ A collection of examples and experiments built using OGL, a lightweight WebGL fr
 
 ## How about some AI chatbot intergration?
 
-### Botpress
+#### Botpress
```
Comment (Dario-Maselli) [inline README.md]: These should all be 3 hashtags. 🙈

**Review by Dario-Maselli — CHANGES_REQUESTED** [review]
You don't need the 4th # for the headings as the 1 level higher heading is 
```md
## How about some AI chatbot intergrations
```

Also, I just noticed that I made a spelling mistake with integrations. 😂

**Comment by Dario-Maselli** [comment]
Maybe it might be a good idea to add links to relevant sites?

File: README.md (hunk 1 above)
Comment (Greenxertz) [reply README.md]: made the changes

**Review by Dario-Maselli — APPROVED** [review]
Looks Good To Me

**Pull request merged**